src/test/java/com/qa/koel/Homework16_Registration.java

```diff
@@ -13,13 +13,12 @@ public class Homework16_Registration extends BaseTest {
     public void registrationNavigationTest() {
         loginPage = new LoginPage(getDriver());
         registrationPage = new RegistrationPage(getDriver());
-        registrationPage.clickLoginFormRegistrationElement();
-        explicitWait.until(ExpectedConditions.visibilityOfElementLocated
-                (By.cssSelector("form>p:nth-child(4)>input[type='email']")));
-        registrationPage.registrationPageEmailField().isDisplayed();
-
-        Assert.assertTrue(registrationPage.getRegisterNewAccountElement());
 
+        registrationPage.clickLoginFormRegistrationElement("//form//div[2]//a")
+//        explicitWait.until(ExpectedConditions.visibilityOfElementLocated
+//                (By.cssSelector("form>p:nth-child(4)>input[type='email']")));
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.

Reply:
I did it because i want it to save them in order to remember what was there before and planned them to remove later. I 'm still in the process of learning and not that smart to remember everything instantly, I wish I could do that ...